src/Notification/Notification.js

```diff
@@ -0,0 +1,351 @@
+		{name: 'client', kind: Control, classes: 'enyo-fill client moon-hspacing'}
+	],
+
+	/**
+	* @private
+	*/
+	bindings: [
+		{from: 'content', to: '$.message.content'},
+		{from: 'allowHtml', to: '$.message.allowHtml'}
+	],
+
+	/**
+	* @private
+	*/
+	create: function () {
+		Popup.prototype.create.apply(this, arguments);
+		this.animateChanged();
+	},
+
+	/**
+	* @private
+	*/
+	render: function () {
+		this._initialized = true;
+		Popup.prototype.render.apply(this, arguments);
+	},
+
+	/**
+	* @private
+	*/
+	showHideDurationChanged: function () {
+		var dur = (this.animate && this.showHideDuration) ? this.showHideDuration : 0;
+		this.set('showingDuration', dur);
+		this.set('hidingDuration', dur);
+		this.applyStyle('-webkit-transition-duration', dur + 'ms');
+		this.applyStyle('-moz-transition-duration', dur + 'ms');
+		this.applyStyle('transition-duration', dur + 'ms');
+	},
+
+	/**
+	* @private
+	*/
+	animateChanged: function () {
+		this.addRemoveClass('animate', this.animate);
+		this.showHideDurationChanged();
+	},
+
+	/**
+	* @private
+	*/
+	showingTransitioningChanged: function () {
+		if (this.showingTransitioning) {
+			// if we are currently animating the hide transition, release
+			// the events captured when popup was initially shown
+			if (this.captureEvents) {
+				this.release();
+			}
+		}
+
+		if (!this.showing) {
+			Spotlight.unspot();
+		}
+	},
+
+	/**
+	* @private
+	*/
+	shown: function (sender, ev) {
+		this.activator = Spotlight.getCurrent();
+
+		var current = this.activator;
+		if (!current || !current.isDescendantOf(this)) {
+			if (Spotlight.isSpottable(this)) {
+				Spotlight.setPointerMode(false);
+				Spotlight.spot(this);
+			}
+			// If we're not spottable, just unspot whatever was previously spotted
+			else {
+				Spotlight.unspot();
+			}
+		}
+
+		if (this.allowBackKey) {
+			this.pushBackHistory();
+		}
+	},
+
+	/**
+	* @private
+	*/
+	hidden: function (sender, ev) {
+		this.respotActivator();
+
+		if (this._initialized && this.allowBackKey && !EnyoHistory.isProcessing()) {
+			EnyoHistory.drop();
+		}
+	},
+
+	/**
+	* Skips animation and jumps to the final shown state.
+	*
+	* @public
+	*/
+	showDirect: function () {
+		var anim = this.animate;
+		if (anim) {
+			this.set('animate', false);
+		}
+		this.show();
+		if (anim) {
+			// getComputedStyleValue forces the browser to update the style rules before
+			// proceeding. Otherwise the removal and addition of the "animate" class happens in
+			// one rendering-pass, which will have no visual difference.
+			this.getComputedStyleValue('display');
+			this.set('animate', anim);
+		}
+	},
+
+	/**
+	* Skips animation and jumps to the final hidden state.
+	*
+	* @public
+	*/
+	hideDirect: function () {
+		var anim = this.animate;
+		if (anim) {
+			this.set('animate', false);
+		}
+		this.hide();
+		if (anim) {
+			this.getComputedStyleValue('display');
+			this.set('animate', anim);
+		}
+	},
+
+	/**
+	* @private
+	*/
+	release: function () {
```

Comment:
Think we can get rid of this, as the implementation is the same as the base kind's.